modules/ROOT/pages/clauses/load-csv.adoc

```diff
@@ -286,115 +449,280 @@ CALL {
-| "/home/example/neo4j/import/artists.csv" |
-+------------------------------------------+
-1 row
++-------------------+
+| No data returned. |
++-------------------+
+Deleted 961 nodes, deleted 372 relationships.
+Removed 2 constraints.
 ----
 
+Note that you can combine multiple queries with a semicolon `;`.
+
+Deletion and creation can be combined into a single process consisting of multiple Cypher queries:
+
+.Query
+[source, cypher]
+----
+MATCH (p:Person) DETACH DELETE p;
+MATCH (m:Movie) DETACH DELETE m;
+
+DROP CONSTRAINT Person_tmdbId IF EXISTS;
+DROP CONSTRAINT Movie_movieId IF EXISTS;
+
+CREATE CONSTRAINT Person_tmdbId IF NOT EXISTS
+FOR (p:Person)
+REQUIRE p.tmdbId IS UNIQUE
+
+CREATE CONSTRAINT Movie_movieId IF NOT EXISTS
+FOR (m:Movie)
+REQUIRE m.movieId IS UNIQUE
+
+LOAD CSV FROM 'https://data.neo4j.com/importing-cypher/persons.csv' AS row
+MERGE (p:Person)
+SET
+p.tmdbId = toInteger(row.tmdbId),
+p.name = row.name,
+p.born = date(row.born);
+
+LOAD CSV FROM 'https://data.neo4j.com/importing-cypher/movies.csv' AS row
+MERGE (m:Movie)
+SET
+m.movieId = toInteger(row.movieId),
+m.title = row.title,
+m.imdbId = toInteger(row.movie_imdbId),
+m.languages = split(row.languages, '|'),
+m.genres = split(row.genres, '|');
+
+LOAD CSV FROM 'https://data.neo4j.com/importing-cypher/acted_in.csv' AS row
+MATCH (p:Person {tmdbId: toInteger(row.person_tmdbId)})
+MATCH (m:Movie {movieId: toInteger(row.movieId)})
+MERGE (p)-[r:ACTED_IN]->(m)
+SET r.role = row.role;
+
+MATCH (p:Person)-[:ACTED_IN]->()
+WITH DISTINCT p SET p:Actor;
+----
+
+.Result
+[role="queryresult"]
+----
++-------------------+
+| No data returned. |
++-------------------+
+Added 2 constraints.
+Nodes created: 961
+Relationships created: 372
+Properties set: 3441
+Labels added: 1065
+----
+
+The example above combines the queries from sections <<load-csv-type-cast-csv-data>>, <<load-csv-split-list-values>>, <<load-csv-create-relationships>>, <<load-csv-create-constraints>> and <<load-csv-create-additional-node-labels>>.
+
+You can run this query at any point to refresh the database with the latest data.
+A single process to build your graph provides a consistent mechanism to test your import.
+
+
+[[load-csv-further-reading]]
+== Further reading
+
+link:https://neo4j.com/docs/getting-started/data-modeling/guide-data-modeling/[Data modeling] considerations are relevant for the data import via `LOAD CSV` as well.
+The imported data may not be optimized for graph database usage and it may be worthwhile to think about what options there are to make full use of Neo4j's feature set.
+
+Furthermore, nodes and relationshops in the resulting graph database can be made more accessible and supportive towards query optimization.
+link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/indexes/[Node indexes] can vastly speed up queries. Also see link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/appendix/tutorials/basic-query-tuning/[Basic query tuning].
```

Comment:
Entities are not _less accessible_ without indexes (you can still access them all), and _supportive towards query optimization_ is a weird phrasing. Also, it's not just _node_ indexes that have an impact. The real fact is that "Creating indexes on frequently queries entities has a huge performance impact on query times." You can phrase it a bit differently, but don't embellish it too much :)